packages/application-generic/src/services/analytics.service.ts

```diff
@@ -48,7 +48,14 @@ export class AnalyticsService {
       };
 
       if (organization.productUseCases) {
-        traits.productUseCases = organization.productUseCases;
+        const productUseCases: string[] = [];
+
+        for (const key in organization.productUseCases) {
+          if (organization.productUseCases[key]) {
+            productUseCases.push(key);
+          }
+        }
```

Comment:
nitpick:

But could also be done like this :)

```suggestion
        const productUseCases: string[] = Object.keys(organization.productUseCases).filter(key => organization.productUseCases[key]);
```

Reply:
It's cleaner, i like it. 🙃